.github/workflows/nodejs.yml

```
@@ -55,3 +55,30 @@ jobs:
           install-command: npm ci --foreground-scripts
       - name: Lint
         run: npm run lint
+
+  validate-lockfile:
+    runs-on: ubuntu-latest
+    env:
+      FORCE_COLOR: 1
+    strategy:
+      matrix:
+        node-version: [18.x]
```

```suggestion
        node-version: [18.x]
        npm-version: [7, 8, 9, 10]
```
(Alternatively: `[7, 8, 9, latest]`

---

`npm-version` is not a thing that `actions/setup-node` supports afaik